src/helix/endpoints/channels/get_vips.rs

```diff
@@ -75,18 +77,10 @@ impl GetVipsRequest {
         self
     }
 
-    /// Filter response with this ID
-    pub fn user_id(self, user_id: impl Into<types::UserId>) -> Self {
-        Self {
-            user_id: vec![user_id.into()],
-            ..self
-        }
-    }
-
```

Comment:
removing this and others is a bit sad, but there is no good way to keep it.

If we could store a `impl IntoIter<Item=impl Into<Cow<'_, Ref>>` wherever we have multiples and have serde properly serialize (and deserialize I guess), that'd be nice.